package.json

```diff
@@ -93,10 +94,12 @@
     "bat-publisher": "^2.0.15",
     "bignumber.js": "^4.0.4",
     "bloodhound-js": "brave/bloodhound",
+    "brave-node-notifier": "brave-intl/brave-node-notifier",
     "clipboard-copy": "^1.0.0",
     "compare-versions": "^3.0.1",
     "date-fns": "^1.29.0",
     "deepmerge": "^2.0.1",
+    "detect-ssid": "^1.0.3",
```

Comment:
why is this needed?

Reply:
to clarify, i see that it's needed to set the current SSID, but i'm wondering why this is a product requirement.

Reply:
the original "ask" from the folks at research was to report the SSID, the IP address and the geolocation for each event. i changed this so that the browser would detect the SSID internally, and allow the user to optionally select a tag from a static list to go along with it. that tag would be reported instead of the three things in the original ask.

the list of tags has entries like "home", "work", "transit", etc., which are both generic and user-selected. i suspect that this is much less a privacy risk than the data requested in the original "ask"...

Reply:
@mrose17 thanks, i will check in with them. is this just for a test group of users, or will it be shipped out in Brave stable?

Reply:
my understanding is that it is only for test groups. when differential privacy is added, perhaps it will return; however, i have no data on that.